MusicTaste/MusicTaste/MusicTaste.xcdatamodeld/MusicTaste.xcdatamodel/contents

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="20086" systemVersion="21F79" minimumToolsVersion="Automatic" sourceLanguage="Objective-C" userDefinedModelVersionIdentifier="">
```

Comment:
Are you saving something to Core Data? Or does the Spotify Framework save something to Core Data?